opentok/opentok-meet
# [OPENTOK-44711] Removing incorrect error messages from the console

#### What is this PR doing?
Removing incorrect error messages from the log.

#### How should this be manually tested?
Run the meet app locally, and join the same session from 2 different tabs, check you see no errors such as `addSubscriber callback` in the console.
To verify the previous error, please perform the same steps on meet.tokbox.com and check you do see those error messages.

#### What are the relevant tickets?
OPENTOK-44711
